suiteData.js

```diff
@@ -32,6 +38,15 @@ class SuiteData {
                 suiteData = await this.getSuiteData(request.query);
                 break;
             case "POST":
+                let tests = request.body?.tests;
+                if (_.isString(tests)) {
+                    const testsDir = path.join(config.testsDir, sanitize(tests));
+                    if (await exists(testsDir)) {
```

Comment:
I don't want to throw if tests isn't string, I want to convert tests to an array of dictionaries, otherwise I assume that tests is an array already (will be validate in the constructor).

Reply:
the comment is on the if exists, not if is string
but it is quite confusing

Reply:
I understand. It's a matter of taste, not a mandatory coding style. I agree that there are cases where it's much more readable to throw at the beginning of a snippet, but this is a very small code snippet and the order of cases doesn't really affect readability.